OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -326,6 +352,9 @@ public override Activity Tick(Actor self)
 					moveFraction = MoveFractionTotal;
 				UpdateCenterLocation(self, Move.mobile);
 
+				if (IsCanceled)
+					ret.NextActivity = Move.NextActivity;
+
```

Comment:
I'm canceling the MoveFirstHalf and MoveSecondHalf because in the render, I'm only drawing the [non canceled activities](https://github.com/Turupawn/OpenRA/blob/visualising_waypoints/OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs#L76).
What do you guys suggest? An extra flag maybe?